application/helpers/update/updates/Update_489.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace LimeSurvey\Helpers\Update;
+
+/**
+ * Some of it is a copy of update 453 which has been skipped on some installations
+ */
+class Update_489 extends DatabaseUpdateBase
+{
+    public function up()
+    {
+        $columnSchema = $this->db->getSchema()->getTable('{{archived_table_settings}}')->getColumn('attributes');
+        if ($columnSchema === null) {
+            $this->db->createCommand()->addColumn('{{archived_table_settings}}', 'attributes', 'text NULL');
+            $archivedTableSettings = \Yii::app()->db->createCommand("SELECT * FROM {{archived_table_settings}}")->queryAll();
+            foreach ($archivedTableSettings as $archivedTableSetting) {
+                if ($archivedTableSetting['tbl_type'] === 'token') {
+                    $this->db->createCommand()->update('{{archived_table_settings}}', ['attributes' => json_encode(['unknown'])], 'id = :id', ['id' => $archivedTableSetting['id']]);
```

Comment:
Why is the field 'attributes' filled with the value 'unknown' ? Is this value actually used somewhere?

Reply:
To be able to fix this for 5.x and 6.x this needs to be put into different DBVersions.
Could you actually see that this was a cause by a bug we released in the past?

Reply:
Let me recall about this and be back :) Thanks!

Reply:
> Why is the field 'attributes' filled with the value 'unknown' ? Is this value actually used somewhere?

Honestly, don;t recall now. Should look further.
That's how it is laid out on the update 450 and 453. As we are re-applying a previous recipe, wanted to keep it consistent.